sql/handler.cc

```diff
@@ -249,12 +249,12 @@ handlerton *ha_default_tmp_handlerton(THD *thd)
 
 /** @brief
   Return the storage engine handlerton for the supplied name
-  
+
   SYNOPSIS
```

Comment:
Please avoid making changes like this:  endspace removal, reformatting. 
Rationale:
* The patch becomes harder to review. First few pages of this diff have nothing to do with ICP or partitioning..
* It clutters revision history.  We have to look at history a lot, and will see that this patch touched this code, then will have to look and see that it didn't make any meaningful changes.

If we have to do this kind of code cleanup, let it be a separate commit with a comment clearly saying it only does reformatting.

Reply:
My apologies, I will fix my editor.  It's configured to trim trailing whitespace on save, but I will make it stop doing that.